src/zope/container/contained.py

```diff
@@ -546,7 +541,7 @@ def setitem(container, setitemf, name, object):
     ...
     TypeError: name not unicode or ascii string
 
-    >>> c = bytes([200]) if PY3 else chr(200)
+    >>> c = bytes([200]) if str is not bytes else chr(200)
```

Comment:
I believe we could spell this `c = b'\xc8'` and it would work on all supported Python versions, as long as this docstring is a raw docstring so the backslash escape doesn't get interpreted before the doctest parser gets to see it.

(Actually we don't need the variable, we can do `b'hello \xc8'` in the setitem() on the next line.)

Reply:
Thanks, I'll simplify.